### ISSUE TITLE
Implement support for .gml files

### ISSUE THREAD

**Issue by ajdapretnar** [biolab/orange3-network#104]
Continuation of https://github.com/biolab/orange3/issues/3608.

##### Network version

1.4.2

##### Orange version

3.21.dev

##### Expected behavior
Network add-on reads and processes .gml files.

##### Actual behavior
Even simple .gml fail.

##### Steps to reproduce the behavior
Load an example data set from https://gephi.org/users/supported-graph-formats/gml-format/.

##### Additional info (worksheets, data, screenshots, ...)
Either discontinue support for .gml (because now it seems as if .gml files are supported) or fix issues with reading .gml files.

**Comment by janezd** [comment]
GML is read by the 3rd-party library networkx.

Heh. We use networkx with its memory-wasting data structure primarily because it can supposedly read different formats. Through networkx, we "support"
- Pajek format, to which we added some patches so that it (mostly) works;
- GML format, to which we would apparently have to add some patches to make it work;
- edgelist, which is trivial (the reader has 15 lines of code)
- pickle, which is, well, just pickle, a single line of code.

I guess it's time to say goodbye to networkx.

Oh, the only other thing we (try to) use networkx for is computation of some graph statistics. They're written in Python and are too slow to be useful, so we'll have to reimplement them anyway (just like we already reimplemented the F-R layout).
